### ISSUE TITLE
Implement `TrackDeleteButton`

### ISSUE THREAD

**Issue by adamnovak** [vgteam/sequenceTubeMap#224]
Someone needs to implement a `TrackDeleteButton` as described in #205. It should have some kind of X or delete icon, an appropriate tool tip/accessibility info, and when clicked call a function passed in from a prop. It ought to work like this:

```
<TrackDeleteButton onClick={function() {alert('Delete!')}} />
```
